bundle/src/main/java/com/adobe/acs/commons/dam/audio/impl/FFMpegAudioUtils.java

```diff
@@ -65,18 +67,11 @@ public static File resolveWorkingDir(String slingHome, String path) {
     }
 
     public static final File createTempDir(File parentDir) throws IOException {
-        File tempDir = null;
-        try {
-            tempDir = File.createTempFile("cqdam", null, parentDir);
-            if (!tempDir.delete()) {
-                throw new IOException("Unable to delete temp directory.");
-            }
-            if (!tempDir.mkdir()) {
-                throw new IOException("Unable to create temp directory.");
-            }
-        } catch (IOException e) {
-            log.warn("could not create temp directory in the [{}] with the exception", parentDir, e);
+        Path tempPath = Files.createTempDirectory(parentDir.toPath(),"cqdam");
+        File tmpDir = tempPath.toFile();
+        if (!tmpDir.mkdir()) {
```

Comment:
I'm not sure why this is necessary -- the call to `Files.createTempDirectory` should actually create the temp directory.